.dockerignore

```diff
@@ -0,0 +1,8 @@
+#.env*
```

Comment:
Rather than allowing these potentially secret files in the docker container, we should pass them through explicitly where necessary (e.g. `docker run --env-file .env.test todo-app:test`), or pass through environment variables individually where we don't have the .env file (e.g. for end-to-end tests, `docker run -e API_KEY -e TOKEN -e .... todo-app:test`)

Reply:
ENV variables already defined in TRAVIS.  Realise I did not need to add file to docker image.